### PR TITLE
[documentation] Add before-after to readme

### PR DESCRIPTION
**Pull request by adsteel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +55,11 @@ Add the following line to your gemfile
 ```ruby
 gem 'dill', github: 'mojotech/dill'
 ```
+
+
+## Contributing
+We welcome contributions! Email Adam at ags@mojotech.com if you're interested.
```
Comment (joelhawksley) [inline README.md]: Might we add a note encouraging pull requests, as this is a public repo?

File: README.md [hunk 2]
```diff
@@ -19,3 +55,11 @@ Add the following line to your gemfile
...
+
+
+## About MojoTech
+We build software that grows businesses. Learn more about us at [our website](www.mojotech.com) or follow us on [our blog](http://blog.mojotech.com/).
```
Comment (joelhawksley) [inline README.md]: Based on how we tag https://github.com/mojotech/jeet, might we add a note about hiring?

**Comment by adsteel** [comment]
@joelhawksley Updated Readme.

**Comment by joelhawksley** [comment]
:+1:

**Comment by aesnyder** [comment]
:+1:

**Pull request merged**